extension/src/extension.ts

```diff
@@ -1,32 +1,35 @@
 import {
-    ExaAllScripts,
+    ExaMetadata,
     ExasolExtension,
     Installation,
     Instance, ParameterValues,
     registerExtension,
     SqlClient
 } from "@exasol/extension-manager-interface";
+import { Context } from "./common";
 import { CONFIG } from "./extension-config";
+import { findInstallations } from "./implementation";
 
 export function createExtension(): ExasolExtension {
     const version = CONFIG.version;
-    const filename = CONFIG.fileName;
```

Comment:
Alternatively we could maybe 
* drop these individual variables 
* put CONFIG.version and .filename immediately into context
* always access filename and version through context: ``context.version``, ``context.filename``